test/functional/rpc_scantxoutset.py

```diff
@@ -20,47 +24,42 @@ def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
+    def sendtodestination(self, destination, amount):
+        # interpret strings as addresses, assume scriptPubKey otherwise
+        if isinstance(destination, str):
+            destination = address_to_scriptpubkey(destination)
+        self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=destination, amount=int(COIN * amount))
 
     def run_test(self):
         self.log.info("Mining blocks...")
-        self.generate(self.nodes[0], 110)
+        self.wallet = MiniWallet(self.nodes[0])
+        self.generate(self.wallet, 120)
```

Comment:
Could this use the pre-mined chain?

Reply:
Good idea. Ironically to use the pre-mined chain MiniWallet needs the call exactly the RPC that this test checks (scantxoutset), but I don't think it is a problem. Done.